.github/workflows/ios.yml

```
@@ -43,8 +43,8 @@ jobs:
     - name: Install targets
       run: make targets-ios
 
-    - name: Set XCode 14.2
-      run: sudo xcode-select -switch /Applications/Xcode_14.2.app
+    - name: Set XCode 15.2
```

Pickiest comment ever:
```suggestion
    - name: Set Xcode 15.2
```